src/internal/tabbable.ts

```diff
@@ -1,14 +1,22 @@
-//
-// This doesn't technically check visibility, it checks if the element has been rendered and can maybe possibly be tabbed
-// to. This is a workaround for shadow roots not having an `offsetParent`.
-//
-// See https://stackoverflow.com/questions/19669786/check-if-element-is-visible-in-dom
-//
-// Previously, we used https://www.npmjs.com/package/composed-offset-position, but recursing up an entire node tree took
-// up a lot of CPU cycles and made focus traps unusable in Chrome / Edge.
-//
-function isTakingUpSpace(elem: HTMLElement): boolean {
-  return Boolean(elem.offsetParent || elem.offsetWidth || elem.offsetHeight || elem.getClientRects().length);
+// Cached compute style calls. This is specifically for browsers that dont support `checkVisibility()`.
+// computedStyle calls are "live" so they only need to be retrieved once for an element.
```

Comment:
Can you explain this more? Do you mean if the coordinates change, the cached reference will be updated _without_ calling `getComputedStyle()`?

Reply:
ComputedStyles are "live" objects, so they auto-update once you have a reference.

> getComputedStyle() returns a live object i.e as the styles of the element are changed the returned object also changes. We don't need to call the function again in order to get the updated CSS styles. It all happens LIVE!

https://www.codeguage.com/courses/js/cssom-computed-styles